modules/ROOT/partials/integrations/react-tech-ref.adoc

```diff
@@ -2,30 +2,31 @@
 
```

Comment:
I can't comment on them, but there are 2 admonitions in this file that need converting properly here (see the `scriptLoading` section).

For example:
```
____
For classic scripts, if the async attribute is present, then the classic script will be fetched in parallel to parsing and evaluated as soon as it is available.
____
```
needs to be converted to:
```
NOTE: For classic scripts, if the async attribute is present, then the classic script will be fetched in parallel to parsing and evaluated as soon as it is available.
```

Reply:
done